docs/vscode-example/tasks.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build debug",
+            "detail": "Build in debug mode",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "build"
+            ]
+        },
+        {
+            "label": "build release",
+            "detail": "Build in release mode",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "build-release"
+            ]
+        },
+        {
+            "label": "clean",
+            "detail": "Clean the target build directory",
+            "type": "process",
+            "group": "build",
+            "command": "cargo",
+            "args": [
+                "make",
+                "clean"
+            ]
+        },
+        {
```

Comment:
i'll change this one to use the cargo-make formatter task.
it comes with built in support to auto install rustfmt